bot/extensions/fun/weather.py

```diff
@@ -0,0 +1,129 @@
+from geopy.geocoders import Nominatim
+from timezonefinder import TimezoneFinder
+import pytz
+from pytz import country_timezones, timezone
+from datetime import datetime
+from discord.ext.commands import Cog, command
+import requests
+from discord import Embed
+import time
+import string
+
+from discord.utils import parse_time
+
```

Comment:
As per the pep8, it is required to have [2 space between top-level functions and classes](https://peps.python.org/pep-0008/#blank-lines).